orangecontrib/network/widgets/OWNxFile.py

```diff
@@ -81,7 +80,7 @@ def reload_data(self):
     def populate_comboboxes(self):
         self.filecombo.clear()
         for file in self.recentFiles or (NONE,):
-            self.filecombo.addItem(path.basename(file))
+            self.filecombo.addItem(path.basename(file[0]))
```

Comment:
Probably due to the changed (PyQt5 compatible) signature of QFileDialog.getOpenFile. If so it is best to change the code at the QFileDialog.getOpenFile [call site](https://github.com/rokgomiscek/orange3-network/blob/a8bebf9d618d2421165b08e77cd4b688e4a5e0d6/orangecontrib/network/widgets/OWNxFile.py#L217) (i.e. `filename, _ = QFileDialog.getOpenFile(...)`. In fact this is necessary judging from the code following it.